UI/obs-app.cpp

```diff
@@ -1919,6 +1919,9 @@ static void main_crash_handler(const char *format, va_list args, void *param)
 	file << text;
 	file.close();
 
+	config_set_bool(GetGlobalConfig(), "Crashes", "PrevCrashed", true);
+	config_save_safe(GetGlobalConfig(), "tmp", nullptr);
+
```

Comment:
If OBS has crashed, it isn't safe to be saving things in the crash handler. Internal state of libobs could be corrupted so this ends up saving a corrupt config or triggering another crash.

Reply:
Changed it to generate txt file that OBS checks on next launch.